analysis.go

```diff
@@ -24,83 +27,22 @@ type Info struct {
 
 // Result is a result of a single check, it can have multiple Event and Info
 // objects attached to it.
+// REVIEW: why do we have YAML annotations?
```

Comment:
I was originally intending to let the analysis output as either YAML or JSON, but eventually decided that was too much effort for little value. I left these values in case we ever returned to the idea.